tests/test_nodes.py

```diff
@@ -163,12 +163,13 @@ def test_ids(self):
         # same query with a $and operator
         npt.assert_equal(_call({"$and": [{Cell.MTYPE: 'L6_Y'}, {Cell.MORPHOLOGY: "morph-B"}]}), [1])
         npt.assert_equal(_call({Cell.MORPHOLOGY: ['morph-A', 'morph-B']}), [0, 1])
+        npt.assert_equal(_call({"$and": [{}, {}]}), [0, 1, 2])
```

Comment:
What is supposed to happen for
```python
_call({"$and": [{Cell.MTYPE: 'L6_Y'}, {}]})
```
Same question with `$or`

Reply:
did it in the next commit